tests/retrieve/icos/test_retrieve_icos.py

```diff
@@ -148,17 +148,37 @@ def test_icos_retrieve_skips_obspack_globalview(mocker, caplog):
 
     # Check attributes within stored Dataset contain extra keys
     attr1_expected_additional = {
-        "measurement_type": "co2 mixing ratio (dry mole fraction)",
+        "species": "co2",
+        "instrument": "FTIR",
+        "instrument_data": ["FTIR", "http://meta.icos-cp.eu/resources/instruments/ATC_505"],
+        "site": "WAO",
+        "measurement_type": "co2 (dry air mole fraction)",
+        "units": "µmol mol-1",
```

Comment:
We're planning to remove "units" from the global attributes at some point in relation to #541 (see also branch rm_units_metadata). We do also need to make sure the units details are converted and stored against the individual species.

Should we remove this in this PR? Or did we want to include it PR but to reference this in the linked issue so we make sure to cover this when fixing it? Thoughts?

Reply:
Yeah I can remove that now.

Reply:
OK let's leave the units attribute as is for now.